registry/handlers/app.go

```diff
@@ -194,6 +195,28 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 		}
 	}
 
+	// configure foreign layers
+	if flConfig, ok := config.Storage["foreignlayers"]; ok {
+		if hosts, ok := flConfig["hostwhitelist"]; ok {
+			switch hosts := hosts.(type) {
+			case []interface{}:
```

Comment:
Is this good as is or should `REGISTRY_STORAGE_FOREIGNLAYER_HOSTWHITELIST='[a.com]` be made to work?